src/veh_interact.cpp

```diff
@@ -572,13 +589,13 @@ void veh_interact::do_install()
     }
     mvwprintz(w_mode, 0, 1, c_ltgray, _("Choose new part to install here:"));
     wrefresh (w_mode);
-    
+
```

Comment:
"Unmergeable pull request" is about conflicting changes that Git doesn't know how to resolve automatically. In this case:

```
<<<<<<< HEAD
    std::array<std::string,7> tab_list = { { pgettext("Vehicle Parts|","All"),
                                             pgettext("Vehicle Parts|","Cargo"),
[...]
    std::vector<vpart_info> tab_vparts = can_mount; // full list of mountable parts, to be filtered according to tab

=======
>>>>>>> origin/pr/10945/head
    int pos = 0;
```

That's right, this one line where your editor automatically removed trailing whitespace is impossible for git to auto-resolve.

To be honest, I'd rather have the trailing space removed -- if everyone did, this would never really happen in the first place -- once a PR that removes it gets merged in, it doesn't happen again for the same line.